setup.py

```diff
@@ -5,37 +5,44 @@
 import distutils.command.sdist
 from distutils.command.install import INSTALL_SCHEMES
 from setuptools import setup, find_packages
-from preup.version import VERSION
+from preupg.version import VERSION
 
 project_name = "preupgrade-assistant"
-project_url = "https://github.com/phracek/preupgrade-assistant/"
+project_url = "https://github.com/upgrades-migrations/preupgrade-assistant/"
 project_author = "Red Hat, Inc."
 project_author_email = "phracek@redhat.com"
 project_description = "Preupgrade assistant"
 package_name = "%s" % project_name
 package_module_name = project_name
 package_version = VERSION
 
-script_files = ['preupg', 'premigrate', 'preup_ui_manage',
-                'preupg-xccdf-compose', 'preupg-create-group-xml',
-                'preupg-content-creator', 'preupg-kickstart-generator']
+script_files = ['bin/preupg', 'bin/premigrate', 'tools/preupg-content-creator',
```

Comment:
Get rid off premigrate.

I would prefer to look on another projects where the binaries are located? In root of the project or in bin directory.

Ask someone.